configuration/src/test/java/io/airlift/configuration/TestConfigurationFactory.java

```diff
@@ -277,6 +281,49 @@ public void testFailedCoercion()
                 ".*Invalid value 'abc %s xyz' for type int \\(property 'int-value'\\).*BeanValidationClass.*");
     }
 
+    @Test
+    public void testSuccessfulFileExistsValidation()
+    {
+        Map<String, String> properties = new HashMap<>();
+        properties.put("file1", "./pom.xml");
+        properties.put("file2", "./pom.xml");
+        properties.put("file3", "./pom.xml");
```

Comment:
use `ImmutableMap.of` just like in the other test

Reply:
This was taken from other test :)